### ISSUE TITLE
Failed to transfer job to other cluster

### ISSUE THREAD

**Issue by scarlett2018** [microsoft/pai#5185]
v1.4 INT bed
Got the following error when transfer job.

![image](https://user-images.githubusercontent.com/39592018/102744247-5548fe80-4394-11eb-841a-5e0d8ce6b723.png)

Expectation
- the error is hard to follow up and fix the issue. we shall provide a better error message for user and admin to identify the failure.

**Comment by hzy46** [comment]
TO DOs for this issue:

- [ ] Add a pre-check to prevent users inputing uri like "https://a.b.c.d/home.html"
- [ ] Do not use openpaisdk but use raw call to check the url

**Comment by mydmdm** [comment]
We should have clear definition (in our code) about the format of uri (e.g., hostname only or a full uri to home page, whether `https://` included, ..) and do a check and converting after user's input. 
> 
> * [ ]  Add a pre-check to prevent users inputing uri like "https://a.b.c.d/home.html"
> * [ ]  Do not use openpaisdk but use raw call to check the url when the user adds bounded clusters
